### ISSUE TITLE
Bad initial resolution.

### ISSUE THREAD

**Issue by AndrewWang996** [tilenskr/Face-Detection-and-Recognition-on-Google-Glass#2]
I believe my Glass is 640 x 360, and the application opens with 640 x 360 resolution, but for whatever reason, it opens up with a distorted fuzzy image. However, the next lower 512 x 288 resolution works fine.

**Comment by tilenskr** [comment]
Yes, Glass resolution is 640 x 360. Some resolutions of camera are indeed blurry. But 640 x 360 resolution should not be blurry. The frame rate of this resolution is not very smooth. Currently I don't posses Google Glass, so I can't test it. But when I get it, I will check on that.

**Comment by AndrewWang996** [comment]
Sorry, I didn't really mean blurry or fuzzy. I meant that the image colors are so distorted that it took me 10 minutes at first to understand that it was even showing a video feed. I think because the resolution is slightly too large, there's some offset created in the rows of pixels and that somehow distorts the feed.

**Comment by tilenskr** [comment]
Ok, thanks for the info. I used default initial resolution, defined in OpenCV and Android Camera API. When I get hands on Glass I will change it.
